### PR TITLE
pages: Add Jetson Orin UEFI instructions

### PR DESCRIPTION
**Pull request by acostach**
Change-type: patch

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

**Comment by acostach** [comment]
To be moved under interact with hardware

File: pages/learn/develop/hardware/jetson-orin.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Jetson Orin
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:2]:
```suggestion
title: Provisioning Jetson Orin
```

File: pages/learn/develop/hardware/jetson-orin.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+# Jetson Orin
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:5]:
```suggestion
# Provisioning Jetson Orin
```

File: pages/learn/develop/hardware/jetson-orin.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## How to check your Jetson Orin's UEFI firmware version
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:7]:
```suggestion
## Check your Jetson Orin's UEFI firmware version
```

File: pages/learn/develop/hardware/jetson-orin.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+You can check which UEFI firmware version your Orin device is running either by connecting to your device's debug UART interface, or by attaching a display to the Orin's video port.
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:9]:
```suggestion
You can check which UEFI firmware version your Jetson Orin device is running either by connecting to your device's debug UART interface, or by attaching a display to the Orin's video port.
```

File: pages/learn/develop/hardware/jetson-orin.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+![Orin UEFI display menu](/img/jetson-orin/jetson_orin_interrupt_booting_uefi.webp)
+
+If the firmware version is older or newer than v36.3, please re-flash its firmware by following the corresponding guide below:
+
+[Firmware update for Orin Nano SD-CARD](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit#firmware)
+[Flashing Jetpack 6 using SDK Manager](https://developer.ridgerun.com/wiki/index.php/JetPack_6_Migration_and_Developer_Guide/Installing_JetPack_6/Flashing_with_SDK_Manager)
+
+## Booting balenaOS flasher images from a USB key
+
+In the unexpected even that your device does not boot the balenaOS flasher image from an attached USB key automatically, please follow these steps:
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:34]:
```suggestion
In the unexpected event that your device does not boot the balenaOS flasher image from an attached USB key automatically, please follow the following steps to make the USB key the boot media to boot from:
```

File: pages/learn/develop/hardware/jetson-orin.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+1) Attach a USB keyboard and a monitor to the device
+2) Power on the device and press 'Esc' when prompted by the UEFI firmware, or 'F11' to enter the Boot Manager Menu directly
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:37]:
```suggestion
1) Attach a USB keyboard and a monitor to the device.
2) Power on the device and press 'Esc' when prompted by the UEFI firmware, or 'F11' to enter the Boot Manager Menu directly.
```

File: pages/learn/develop/hardware/jetson-orin.md (hunk 2 above)
Comment (acostach) [reply pages/learn/develop/hardware/jetson-orin.md]: thanks Vipul, suggestion is commited

File: pages/learn/develop/hardware/jetson-orin.md (hunk 3 above)
Comment (acostach) [reply pages/learn/develop/hardware/jetson-orin.md]: suggestion commited

File: pages/learn/develop/hardware/jetson-orin.md (hunk 4 above)
Comment (acostach) [reply pages/learn/develop/hardware/jetson-orin.md]: change committed

File: pages/learn/develop/hardware/jetson-orin.md (hunk 5 above)
Comment (acostach) [reply pages/learn/develop/hardware/jetson-orin.md]: done

File: pages/learn/develop/hardware/jetson-orin.md (hunk 6 above)
Comment (acostach) [reply pages/learn/develop/hardware/jetson-orin.md]: applied

File: pages/learn/develop/hardware/jetson-orin.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+![Orin UEFI uart menu](/img/jetson-orin/jetson_orin_uart_uefi_menu.webp)
+
+If your Orin is connected to a monitor, the UEFI firmware version will be displayed at the top of the screen:
+
+![Orin UEFI display menu](/img/jetson-orin/jetson_orin_interrupt_booting_uefi.webp)
+
+If the firmware version is older or newer than v36.3, please re-flash its firmware by following the corresponding guide below:
+
+[Firmware update for Orin Nano SD-CARD](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit#firmware)
+[Flashing Jetpack 6 using SDK Manager](https://developer.ridgerun.com/wiki/index.php/JetPack_6_Migration_and_Developer_Guide/Installing_JetPack_6/Flashing_with_SDK_Manager)
```
Comment (vipulgupta2048) [inline pages/learn/develop/hardware/jetson-orin.md:30]:
```suggestion
- [Firmware update for Orin Nano SD-CARD](https://developer.nvidia.com/embedded/learn/get-started-jetson-orin-nano-devkit#firmware)
- [Flashing Jetpack 6 using SDK Manager](https://developer.ridgerun.com/wiki/index.php/JetPack_6_Migration_and_Developer_Guide/Installing_JetPack_6/Flashing_with_SDK_Manager)
```

**Review by vipulgupta2048 — COMMENTED** [review]
LGTM

**Review by vipulgupta2048 — APPROVED** [review]

**Review by myarmolinsky — APPROVED** [review]

**Pull request merged**